packages/@aws-cdk/aws-rds/test/serverless/integ.serverless-cluster.ts

```diff
@@ -0,0 +1,35 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as cdk from '@aws-cdk/core';
+import { DatabaseClusterEngine, ParameterGroup, ServerlessDatabaseCluster } from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-rds-integ');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 2 });
```

Comment:
I would go `maxAzs: 1`, makes the deploy faster 🙂.

Reply:
we have validation that would fail if we drop below 2 AZs. left this at 1